Certification-Standard/introduction.md

```diff
@@ -0,0 +1,239 @@
+
+The Cardano Certification Standard also serves as a guide for both DApp developers and evaluators. It offers insights and best practices for the development of DApps with robust security features and provides a structured approach for evaluating their security posture.
+
+Flexibility is a core principle of the Cardano Certification Standard. It is designed to accommodate a diverse range of evaluation methodologies, ensuring that all essential security properties required by a DApp can be effectively assessed. However, it is essential to note that the validity of an evaluation is contingent upon the specific security properties evaluated and the methods employed. Auditors and users are advised to examine the DApp's security requirements and evaluation methodologies to determine the meaningfulness of the evaluation results.
+
+The Cardano Certification Standard aims at adressing:
+
+- The protection of assets of any user interacting with the DApp
+- The loss of use of the DApp
+- ...
+
+Moreover, the scope of the Cardano Certification Standard could extend to security aspects beyond these categories.
+
+Nevertheless, it is important to acknowledge that certain topics are explicitly excluded from the purview of the Cardano Certification Standard.
+
+Among these exclusions are:
+- ...
+
+In conclusion, the Cardano Certification Standard serves as a document promoting transparency, consistency, and confidence in the security of DApps.
+
+*RSoulat* This document draws heavily from the work of Common Criteria. I think we should take some inspiration from it for the security aspects, and, if we want to deal with purely functional aspects, add them where we see fit.
+
+## 2 General model
+
+### 2.1 Scope
+
+The CCS wishes to establish the general concepts and principles of DApp security evaluation and to specify the general model of evauation. This section provides an overview of all the parts of the CCS, it defines all the different terms, abbreviations and concepts used through the rest of the document. It establishes the core principles of the standard:
+
+- Protection Profiles
+- Security Targets
+- Target of Evaluation
+- Evaluation Levels
+
+### 2.2 Terms and Definitions
+
+*RSoulat* Note: Taken from the CC definitions
+*RSoulat* Maybe we could drop the Protection Profiles from a first version and only deal with Security Target
+
+**Adverse action**: An action performed by a Theat agent on an Asset
+
+**Asset**: Entity that the owner of the Target of Evaluation places value on.
+*RSoulat* Example of assets we could consider: Ada, Tokens, NFTs, Wallet private keys
+
+**Attack potential**: Measure of the effort needed to exploit a vulnerability in a Targer of Evaluation (TOE)
+
+**Attack Surface**: Set of logical interfaces to a target, consisting of points through which access to the target's assets or control.
```

Comment:
Set of logical interfaces to a target, which provide access to the target's assets or control.